src/common/hooks/useHoldingsAndLocations/useHoldingsAndLocations.js

```diff
@@ -1,23 +1,25 @@
 import { useQuery } from 'react-query';
 
-import {
-  batchRequest,
-  HOLDINGS_API,
-  LIMIT_MAX,
-  LOCATIONS_API,
-} from '@folio/stripes-acq-components';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
 import {
   useNamespace,
   useOkapiKy,
 } from '@folio/stripes/core';
 
+import {
+  getHoldingLocations,
+  getHoldingLocationsByTenants,
+} from '../../utils/getHoldingLocations';
+
 const DEFAULT_DATA = [];
 
-export const useHoldingsAndLocations = ({ instanceId, tenantId, options = {} }) => {
-  const {
-    enabled = true,
-    ...queryOptions
-  } = options;
+export const useHoldingsAndLocations = ({
+  instanceId,
+  options = {},
+  tenantId,
+  tenants = [],
```

Comment:
Could you please add comments in the code on what the `tenantId` and `tenants` props are used for?

Reply:
Thanks for the explanation, @alisher-epam, that is very helpful context. Please include it in a comment in the code as was requested. It will really simplify future debugging if we don't have to find the PR in GitHub in order to understand the purpose of something that is otherwise undocumented in the codebase and the commit itself.